src/main.c

```diff
@@ -30,23 +61,45 @@ print_error(char *prefix, rustls_result result)
   fprintf(stderr, "%s: %.*s\n", prefix, (int)n, buf);
 }
 
+#ifdef _WIN32
+const char *ws_strerror (int err)
+{
+  static char ws_err[50];
+
+  if (err >= WSABASEERR) {
+    snprintf(ws_err, sizeof(ws_err), "Winsock err: %d", err);
+    return ws_err;
+  }
+  /* Assume a CRT error */
+  return (strerror)(err);
```

Comment:
Why is strerror in parentheses here?

Reply:
To get the original CRT `strerror()` back after I changed it.